frontend/src/utils/fetch-with-timeout.ts

```diff
@@ -8,7 +8,7 @@ interface FetchWithTimeoutOptions extends RequestInit {
 
 export const ANALYSIS_REQUEST_TIMEOUT = 60000;
 
-const DEFAULT_REQUEST_TIMEOUT = 15000;
+const DEFAULT_REQUEST_TIMEOUT = 30000;
```

Comment:
Still needed? Is it to account for the retries?

Reply:
It's still needed. Not necessarily for retries but for loading all of the data needed for RBD. The server needs to make a lot of separate requests to get all of the various country data and, even with parallelization, it can take about 20 seconds.